src/store/slices/node/actionsAsync.ts

```diff
@@ -59,93 +81,207 @@ const {
   withdraw,
 } = api;
 
-const getInfoThunk = createAsyncThunk('node/getInfo', async (payload: BasePayloadType, { rejectWithValue }) => {
-  try {
-    const info = await getInfo(payload);
-    return info;
-  } catch (e) {
-    if (e instanceof APIError) {
-      return rejectWithValue({
-        status: e.status,
-        error: e.error,
-      });
+const getInfoThunk = createAsyncThunk<GetInfoResponseType | undefined, BasePayloadType, { state: RootState }>(
+  'node/getInfo',
+  async (payload: BasePayloadType, {
+    rejectWithValue,
+    dispatch,
+  }) => {
+    dispatch(setInfoFetching(true));
+    try {
+      const info = await getInfo(payload);
+      return info;
+    } catch (e) {
+      if (e instanceof APIError) {
+        return rejectWithValue({
+          status: e.status,
+          error: e.error,
+        });
+      }
     }
-  }
-});
+  },
+  { condition: (_payload, { getState }) => {
+    const isFetching = getState().node.info.isFetching;
+    console.log('getInfo isFetching:', isFetching);
```

Comment:
i think we should remove this console log

Reply:
Comments removed from node and safe actionsAsync